### PR TITLE
Fix all linter warnings

### PR DESCRIPTION
**Pull request by V02460**
Signed-off-by: Kai A. Hiller <V02460@gmail.com>

### CODE REVIEW

File: lib/components/prometheusmetrics.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ function PrometheusMetrics() {
 
     // prom-client 9.0.0 has to be asked explicitly; previous versions would
     //   do this by default
-    if (client.collectDefaultMetrics) client.collectDefaultMetrics();
+    if (client.collectDefaultMetrics) {client.collectDefaultMetrics();}
```
Comment (Half-Shot) [inline lib/components/prometheusmetrics.js]: I guess we need to rule this, but I'd rather we did:

```
if (thing) {
 dodathing()
}
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ describe("RoomLinkValidator", () => {
             let joined;
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { joined = room_id; return Promise.resolve(); },
+                    join: (roomID) => { joined = roomID; return Promise.resolve(); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { joined = roomId; return Promise.resolve(); },
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ describe("RoomLinkValidator", () => {
             let joined;
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { joined = room_id; return Promise.reject({errcode: "M_FORBIDDEN"}); },
+                    join: (roomID) => { joined = roomID; return Promise.reject({errcode: "M_FORBIDDEN"}); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { joined = roomId; return Promise.reject({errcode: "M_FORBIDDEN"}); },
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 4]
```diff
@@ -55,7 +55,7 @@ describe("RoomLinkValidator", () => {
             let joined;
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { joined = room_id; return Promise.reject({}); },
+                    join: (roomID) => { joined = roomID; return Promise.reject({}); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { joined = roomId; return Promise.reject({}); },
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 5]
```diff
@@ -78,7 +78,7 @@ describe("RoomLinkValidator", () => {
             let joined;
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { joined = room_id; return Promise.resolve({}); },
+                    join: (roomID) => { joined = roomID; return Promise.resolve({}); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { joined = roomId; return Promise.resolve({}); },
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 6]
```diff
@@ -91,7 +91,7 @@ describe("RoomLinkValidator", () => {
             let joined;
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { joined = room_id; return Promise.reject({errcode: "M_FORBIDDEN"}); },
+                    join: (roomID) => { joined = roomID; return Promise.reject({errcode: "M_FORBIDDEN"}); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { joined = roomId; return Promise.reject({errcode: "M_FORBIDDEN"}); },
```

File: spec/unit/room-upgrade-handler.spec.js [hunk 7]
```diff
@@ -103,7 +103,7 @@ describe("RoomLinkValidator", () => {
         it("should fail for any other reason", () => {
             const bridge = {
                 getIntent: () => ({
-                    join: (room_id) => { return Promise.reject({}); },
+                    join: (roomID) => { return Promise.reject({}); },
```
Comment (Half-Shot) [inline spec/unit/room-upgrade-handler.spec.js]:
```suggestion
                    join: (roomId) => { return Promise.reject({}); },
```

File: lib/components/prometheusmetrics.js (hunk 1 above)
Comment (V02460) [reply lib/components/prometheusmetrics.js]:
```suggestion
    if (client.collectDefaultMetrics) {
        client.collectDefaultMetrics();
    }
```

**Pull request merged**